_posts/2016-03-28-ember-conf-day-1.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "EmberConf Recap -- Day 1"
+description: "A short recap of some of the presentations at EmberConf -- Day 1"
+category:
+tags: [ember, ember.js, conference, emberconf]
+author: vikram_ramakrishnan
+---
+{% include JB/setup %}
+
+88.89% of [Greenfield](www.greenfieldhq.com)'s team is attending [EmberConf 2016](http://emberconf.com/schedule.html)!
+With over 950+ attendees, this EmberConf's largest showing yet. It's an
+exciting time to be working in Ember, and this short blog post highlights 
+some of today's presentations:
+
+###Keynote
+This was a really great keynote by [Yehuda Katz](https://twitter.com/wycats) & [Tom Dale](https://twitter.com/tomdale) highlight where Ember is headed, with a particular focus on performance & optimization.
+
+- Ember core team remains but is also dividing up into sub-teams (Ember Data, Ember-CLI, Learning)
+- The Ember core team is really pushing to be more mobile friendly
```

Comment:
Maybe elaborate a bit more? Sounded at first like maybe the core team wants to be more remote. Instead, I think this is a focus on Ember performance on mobile devices. Especially Android. See https://eviltrout.com/2016/02/25/fixing-android-performance.html